tests/unit/test_models/test_base_model.py

```diff
@@ -385,18 +373,18 @@ def test_print_parameter_info_submodel(self):
         sys.stdout = sys.__stdout__
 
         result = captured_output.getvalue().strip()
-        self.assertIn("'sub1' submodel parameters:", result)
-        self.assertIn("'sub2' submodel parameters:", result)
-        self.assertIn("Parameter", result)
-        self.assertIn("InputParameter", result)
-        self.assertIn("FunctionParameter with inputs(s) ''", result)
-        self.assertIn("InputParameter in ['test']", result)
-        self.assertIn("g", result)
-        self.assertIn("a", result)
-        self.assertIn("c", result)
-        self.assertIn("h", result)
-        self.assertIn("b", result)
-        self.assertIn("d", result)
+        assert "'sub1' submodel parameters:" in result
+        assert "'sub2' submodel parameters:" in result
+        assert "Parameter" in result
+        assert "InputParameter" in result
+        assert "FunctionParameter with inputs(s) ''" in result
+        assert "InputParameter in ['test']" in result
+        assert "g" in result
+        assert "a" in result
+        assert "c" in result
+        assert "h" in result
+        assert "b" in result
+        assert "d" in result
```

Comment:
This can be parameterized as well. I would recommend opening a subsequent PR that implements nothing but parameterization (and similarly for fixtures, etc.) and changing this test there.